src/FixedDataTable.js

```diff
@@ -1084,12 +1118,12 @@ class FixedDataTable extends React.Component {
       onScrollStart(oldScrollX, oldScrollY, oldFirstRowIndex)
     }
 
-    if (scrollXChanged && onHorizontalScroll) {
-      onHorizontalScroll(scrollX);
+    if (scrollXChanged) {
+      this._scrollToX(scrollX);
```

Comment:
This (and below) doesn't seem to work for controlled scrolling... `onVerticalScroll` and `onHorizontalScroll` is not being called.

I think the issue is that setting the internal state with this._scrollbarYRef.scrollTop(y) doesn't trigger a callback. See https://github.com/malte-wessel/react-custom-scrollbars/blob/master/src/Scrollbars/index.js#L97-L211

I'm not even sure if it renders the scroll bar with the updated values? Can you investigate this?